hermetic_build/README.md

```diff
@@ -25,13 +25,20 @@ for more information.
 
 ## Parameters to generate a repository using the docker image
 
-### Generation configuration yaml (`generation-config-path`)
+### Generation configuration yaml (`generation-config-path`), optional
```

Comment:
Can we add a sentence before this section mentioning something like "All the parameters are optional, but a generation_config.yaml is required"?

Reply:
I added this to `Prerequisites`.